support/build.gradle

```diff
@@ -1,8 +1,20 @@
+def VERSION_NAME = '0.1.0'
```

Comment:
One thing to keep in mind is that bintray will reject your upload if the version matches what's already been upload.
So, you can only upload 0.1.0 once.